[DOCS] Update detection rule exceptions export and import (#1434)
elastic/security-docs: docs/detections/detections-ui-exceptions.asciidoc

If no attribute is selected, the app searches the list name by default.

To export or delete an exception list, select the required action button on the appropriate list. Note the following:

> * Exception lists are exported to `.ndjson` format.

Reviewer: ```suggestion
* Exception lists are exported to `.ndjson` files.
```

> * Exception lists are also exported when you export any detection rules configured with exceptions. See <<import-export-rules-ui>>.

Reviewer: ```suggestion
* Exception lists are also exported as part of any exported detection rules configured with exceptions. See <<import-export-rules-ui>>.
```
